public/views/answers.html

```diff
@@ -42,7 +43,7 @@
           <h3>How To Play</h3>
         </div>
         <div class="container">
-          <ol id="oh-el" class="game-instructions">
+          <ol id="oh-el" class="justify">
```

Comment:
Why did you change game instructions class

Reply:
The styling didn't look go enough. I got a suggestion from a team mate and that suggestion was what I implemented.